docs/quick-ingestion-guides/snowflake/overview.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Overview
+---
+# Snowflake Ingestion Guide: Overview
+
+## What You Will Get Out of This Guide
+
+This guide will help you set up the Snowflake connector through the DataHub UI to begin ingesting metadata into DataHub.
+
+Upon completing this guide, you will have a recurring ingestion pipeline that will extract metadata from Snowflake and load it into DataHub. This will include to following Snowflake asset types:
+
+* Databases
+* Schemas
+* Tables
+* External Tables
+* Views
+* Materialized Views
+
+This recurring ingestion pipeline will also extract:
+
+* **Usage statistics** to help you understand recent query activity (available if using Snowflake Enterprise edition or above)
+* **Table- and Column-level lineage** to automatically define interdependencies between datasets and columns (available if using Snowflake Enterprise edition or above)
+* **Table- and column-level profile statistics** to help you understand the shape of the data
```

Comment:
Column level profiling is disabled by default. So probably change this to **Table-level profile statistics** ?